simulator/src/errors.py

```diff
@@ -0,0 +1,2 @@
+class ValidationError(Exception):
+    pass
```

Comment:
Add docstring:

"""Raised when errors are encountered in cofig file validation"""